commerce_coordinator/apps/iap/api/v1/utils.py

```diff
@@ -135,3 +136,26 @@ def get_standalone_price_for_sku(sku: str) -> Money:
         )
         logger.exception(message, exc_info=exc)
         raise ValueError(message) from exc
+
+
+def get_payment_processor(request, cart, payment_processor: str):
```

Comment:
we can just pass request_data, and get course_run_key, payment_processor, and purchase_token from it
```suggestion
def get_payment_data(request_data):
```